compiler/codegen/FrontEnd.hpp

```diff
@@ -152,11 +152,9 @@ class TR_FrontEnd : public TR_Uncopyable
    // Needs VMThread
 
    // In practice we never enable methodEnterTracing without methodExitTracing.
-   // Combine them into one method: isMethodTracingEnabled() which should be used
-   // to replace all usages of isMethodEnterTracingEnabled() and isMethodExitTracingEnabled().
+   // isMethodEnterTracingEnabled() and isMethodExitTracingEnabled() are replaced
+   // by isMethodTracingEnabled().
```

Comment:
I don't think this comment is relevant. Can it be replaced with a description of what the function does?